src/containers/MainContainer/MainContainer.js

```diff
@@ -12,31 +14,72 @@ class MainContainer extends Component {
         super(props)
         this.props = props
         this.state = {
-            posts: []
+            posts: [],
+            tabIndex: 0,
+            tabs: ['Active Disasters', 'Resources', 'FEMA Checklist'],
         }
     }
 
     componentDidMount() {
-        return getPosts().then(result => {
+        return this.getAllActivePosts()
+    }
+
+    getAllActivePosts = () => {
+        getPosts().then(result => {
             this.setState(previousState => {
                 return {
                     ...previousState,
-                    posts: result
+                    posts: result.length > 0 ? result : []
```

Comment:
Why is the conditional here needed?  Assuming `getPosts` always returns an array.

Reply:
Mainly just for safe checking but will investigate if requires another refactor